app/src/main/res/values/arrays.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
If you're not using this file or retrieving anything from this file then it's useless and you can remove it and whenever you need it in future then just create it again